src/shared/components/organisms/Settings/atoms/SettingSidenav.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from '../../../../lib/styled'
```

Comment:
For small one time use components like this. I would prefer to just keep those in the organism index directly
Something like

settings/index.tsx
```
 return <Container>
    <DoublePane className='settings' right={<div className='settings__content'>{settingsContent}</div>}
<nav className ='settings__nav'>{settingsNav}</nav>}
</DoublePane> 
</Container>
```

edit: Although I understand why you did it this way, in my opinion `SettingsLayout` should not expect a `SettingsSideNav` as props. Instead it should put the react.Node it receives into the `SettingsSideNav` instead, because that's the expected behaviour. There should be no other case where we want our settings' sidenav in our layout to be anything else styling wise. 

But it might need a bit more refactoring for this to happen so I think it's fine for now, I will fix it by myself. 
Just keep in mind for the future